html/docs/include/functions/xdebug_start_gcstats

```diff
@@ -0,0 +1,26 @@
+= xdebug_start_gcstats
```

Comment:
Obscure feature: add ``,2.6`` at the end of the function name to show in which version this is introduced.